stdlib/2/typing.pyi

```diff
@@ -457,7 +457,8 @@ def cast(tp: str, obj: Any) -> Any: ...
 class NamedTuple(tuple):
     _fields = ...  # type: Tuple[str, ...]
 
-    def __init__(self, typename: str, fields: Iterable[Tuple[str, Any]] = ..., *,
+    def __init__(self, typename: Union[str, Text],
+                 fields: Iterable[Tuple[Union[str, Text], Any]] = ..., *,
```

Comment:
Thank you, but you should use just `Text` instead of `Union[str, unicode]` or `Union[str, Text]`. `Text` in argument types is equivalent to both `bytes` and `str` (per PEP 484).

Reply:
Ah, I was mislead by this excerpt from PEP 484:
> Text, a simple alias for str in Python 3, for unicode in Python 2

Also, in the very same file (`stdlib/2/typing.pyi`) there is this line:
```python
Text = unicode
```

But now I see that PEP 484 also states:
> For parameters typed as Text, arguments of type str as well as unicode should be acceptable.

Should be good now.